secretmanager/snippets/regional_samples/create_regional_secret.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+Command line application and sample code for creating a new regional secret.
+"""
+
+import argparse
+from typing import Optional
+
+from google.cloud import secretmanager_v1
```

Comment:
Just noticed this now - should the imports be right underneath the region tag (line 25~26) instead, and remove this one and on line 37?